pkg/ocm/ocm.go

```diff
@@ -43,14 +43,27 @@ func New(ocmConfigFile string) (Client, error) {
 	var err error
 	client := Client{}
 
-	cfg, err := newConfigFromFile(ocmConfigFile)
-	if err != nil {
-		return client, fmt.Errorf("failed to load config file: %w", err)
+	ocmClientID, hasOcmClientID := os.LookupEnv("OCM_CLIENT_ID")
```

Comment:
I chose to omit the prefix of `CAD` on the other envvars as we had a client that used the previous envvars.

If we are unique on using this we should prefix with a CAD to signify this is unique to this code.

Once we decide this will be generally used we can omit the prefix and push upstream

Reply:
I agree.. naming the env vars the same like in the vault makes sense